### PR TITLE
DCOS-39069: use headercell from ui-kit

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
## Testing

Change this line in `NodesTableContainer.js`

```diff
-import NodesTable from "../../../components/NodesTable";
+import NodesTable from "../../../components/NodesTable-new";
```

After that please take a look at the nodes table and see if the headlines of the table look correctly

## Trade-offs

- The content spacing is a separate issue (DCOS-39265), therefore it looks a bit off

## Dependencies

None

### CODE REVIEW

**Comment by juliangieseke** [comment]
@DanielMSchmidt CI is red with the build issue again 😩 maybe you can try to cut a new release and use that one…

**Comment by juliangieseke** [comment]
Oh, and I have the same issue on my machine 😭

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "@dcos/http-service": "0.3.0",
     "@dcos/mesos-client": "0.2.3",
     "@dcos/tslint-config": "0.1.0",
-    "@dcos/ui-kit": "1.1.1",
+    "@dcos/ui-kit": "1.2.2",
```
[the remark this answers is not in the document]
Comment (DanielMSchmidt) [reply package.json]: I'll update real quick 💪

**Comment by nLight** [comment]
ui-lik@1.2.3 doesn't want to compile. ts-lint errors

**Comment by DanielMSchmidt** [comment]
@weblancaster Do you have an idea why the ui-kit doesn't compile? 🤔

**Comment by juliangieseke** [comment]
@weblancaster @DanielMSchmidt I released ui-kit 1.2.4, 1.2.3 imports looked different to the ones from ui-kit master… Can you please try using that one? 🙇

**Comment by juliangieseke** [comment]
1.2.4 is also broken :( cc @Fabs

**Comment by DanielMSchmidt** [comment]
Hey Team, I had no time to double check this, but I am 90% confident that it worked. If any of you can spare the time it would be great if you could check that @juliangieseke issue is gone now. If not I will do it myself tomorrow :) Thank you for your help 👍

**Pull request merged**